examples/glEvalLimit/glEvalLimit.cpp

```diff
@@ -1397,12 +1408,28 @@ initHUD() {
     g_hud.AddPullDownButton(shading_pulldown, "FaceVarying", kFACEVARYING, g_drawMode==kFACEVARYING);
     g_hud.AddPullDownButton(shading_pulldown, "Mean Curvature", kMEAN_CURVATURE, g_drawMode==kMEAN_CURVATURE);
 
-    g_hud.AddCheckBox("Adaptive (`)", g_adaptive != 0, 10, 190, callbackCheckBox, kHUD_CB_ADAPTIVE, '`');
+    g_hud.AddCheckBox("Adaptive (`)", g_adaptive != 0, 10, 150, callbackCheckBox, kHUD_CB_ADAPTIVE, '`');
+
+    g_hud.AddCheckBox("Smooth Corner Patch (O)", g_smoothCornerPatch!=0,
+                      10, 170, callbackCheckBox, kHUD_CB_SMOOTH_CORNER_PATCH, 'o');
+//  g_hud.AddCheckBox("Single Crease Patch (S)", g_singleCreasePatch!=0,
+//                    10, 190, callbackCheckBox, kHUD_CB_SINGLE_CREASE_PATCH, 's');
```

Comment:
Is this commented-out code intentional?

Reply:
Commenting this out makes sense for glEvalLimit since single crease patches are currently supported for drawing only (not yet via the patch evaluation code paths).

The rest of the changes look good to me. Thanks, Barry!